### PR TITLE
docs: do not recommend PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD anymore

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/intro-js.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ npx playwright --version
 
 ## System requirements
 
-- Node.js 16+
+- Node.js 18+
```
Comment (mxschmitt) [inline docs/src/intro-js.md:127]: drive-by

**Review by yury-s — APPROVED** [review]
